tgui/packages/tgui/interfaces/BluespaceTap.js

```diff
@@ -153,3 +172,38 @@ export const BluespaceTap = (props, context) => {
     </Window>
   );
 };
+
+export const Alerts = (props, context) => {
```

Comment:
this is great work 👍

Reply:
Thanks!
 Is there anything that needs doing or can I resolve this?

Reply:
can resolve